package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "node server",
     "start:dev": "node server --dev",
-    "start:poll": "node server --dev --watch-poll",
+    "start:poll": "node --max_old_space_size=4096 server --dev --watch-poll",
```

Comment:
Where you having issues with `webpack` crashing the node server?

Reply:
Yep! Unfortunately. With bcash and hsd libraries now included there is a lot more webpack is keeping in memory and it would overflow without this.